src/content/docs/it/tutorial/1-setup/1.mdx

```diff
@@ -0,0 +1,118 @@
+---
+type: tutorial
+title: Prepara il tuo ambiente di sviluppo
+description: |-
+  Tutorial: Costruisci il tuo primo blog con Astro —
+  Installa sul tuo computer gli strumenti di cui avrai bisogno per completare il tutorial
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Installa tutti gli strumenti di cui avrai bisogno per costruire il tuo sito Astro
+</PreCheck>
+
+## Come ottenere gli strumenti di sviluppo necessari
+
+### Terminale
+
+Userai un **terminale a linea di comando** per creare il tuo progetto Astro e per eseguire i comandi più importanti per assemblare, sviluppare e testare il tuo sito.
+
+Potrai accedere alla linea di comando usando un programma cosiddetto "Terminale" per il tuo sistema operativo. I terminali più diffusi sono **Terminale** (MacOS/Linux), **Prompt dei comandi** (Windows) o **Termux** (Android). Probabilmente uno di questi è già presente sulla tua macchina.
```

Comment:
Non sono sicuro la scriverei in questo modo, personalmente. Probabilmente eviterei il "cosiddetto".

```suggestion
Potrai accedere alla linea di comando usando un programma terminale locale per il tuo sistema operativo. I terminali più diffusi sono **Terminale** (MacOS/Linux), **Prompt dei comandi** (Windows) o **Termux** (Android). Probabilmente uno di questi è già presente sulla tua macchina.
```